lib/unique_elements.rb

```diff
@@ -2,5 +2,15 @@
 # combination of the above), and returns an array of the elements that appear once and only
 # once.
 def find_unique_elements(arr)
-  
+
+arr = [1, 2, "three", "three", 4]
```

Comment:
Same issue here as on the last problem. Every time you run this method, you are operating on the array `[1, 2, "three", "three", 4]` instead of the array that the user (or the testing module) passes to the method.